src/Command/TailwindBuildCommand.php

```diff
@@ -32,6 +33,7 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addArgument('input', InputArgument::OPTIONAL, 'The input CSS file to compile')
```

Comment:
Should we really care about specifying the input file on running the command? Why can't we just build both files (I mean, all the input CSS files listed in the `input_css` option)? It will simplify things, no need to remember to pass the argument. Because as I understand, if you miss this arg (which is optional) - only the first entry on the `input_css` will be built. I'm afraid it will lead to WTF moments. IMO we should build all input CSS files listed by default, though I'm not against of keeping that optional arg in case you want to build a specific input CSS file only.

And how about to be consistent and call this `input_css` to match the config option? Does it make sense to you?

Reply:
It is possible to build both files, but since the Tailwind binary doesn't natively support this, we will need a workaround:
- In default mode (without the `--watch` flag), it would run the process sequentially for each input file (or possibly in parallel if feasible).
- In watch mode (with the `--watch` flag), we would need to spawn as many processes as there are input files. I'm not sure if this will be efficient in terms of memory usage. I'll look into potential improvements.

I also think we can keep `input_css` as the argument name. I initially used `input` because in the first implementation, I wanted to provide it as a flag. I'll rename it back to `input_css`.

Reply:
I see, then the current implementation should be good IMO to keep things simple. But if anyone have any ideas how to improve it even further - feel free to share

Reply:
Great, thanks for your feedback.
I'll maybe try to improve this in another PR.